src/test/java/com/github/iveina2021/api_test/DeleteAccountTest.java

```diff
@@ -0,0 +1,33 @@
+package com.github.iveina2021.api_test;
+
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.Map;
+
+public class DeleteAccountTest {
+    @BeforeEach
+    public void setUp() {
+        RestAssured.baseURI = "https://automationexercise.com";
```

Comment:
(minor) you may extract **https://automationexercise.com** to constants to reuse it everywhere.

Reply:
Made a constant BASE_URL and reuse it everywhere